tests/preprocessing/test_positionfixes.py

```diff
@@ -123,6 +137,17 @@ def test_temporal(self):
 class TestGenerate_triplegs:
     """Tests for generate_triplegs() method."""
 
+    def test_duplicate_columns(self, geolife_pfs_stps_long):
+        """Test if running the function twice, the generated column does not yield exception in join statement"""
+
+        # we run generate_triplegs twice in order to check that the extra column (tripleg_id) does
+        # not cause any problems in the second run
+        pfs, stps = geolife_pfs_stps_long
+        # case 1
+        pfs_run_1, tpls = pfs.as_positionfixes.generate_triplegs(stps, method="between_staypoints")
+        pfs_run_2, tpls = pfs_run_1.as_positionfixes.generate_triplegs(stps, method="between_staypoints")
```

Comment:
Line 147-148, `tpls` is not used, replace with `_`

Reply:
done